tests/stdlib/trandom.nim

```diff
@@ -1,21 +1,18 @@
 discard """
-  action: compile
+  joinable: false
 """
 
 import random
 
+randomize(233)
+
 proc main =
   var occur: array[1000, int]
 
   var x = 8234
   for i in 0..100_000:
     x = rand(high(occur))
     inc occur[x]
-  for i, oc in occur:
```

Comment:
we should keep this test and adjust it to make it pass, it prevents regressions in case distribution becomes non-uniform. This works:
```nim
  var occur: array[1000, int]
  for i in 0..<100_000:
    var x = rand(high(occur))
    inc occur[x]
  # echo (min(occur), max(occur))
  doAssert max(occur) <= 140 and min(occur) >= 60 # gives some slack
```